lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -207,17 +213,17 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
       if (llvm::count(outputShape, kUnknownSize) != 1 ||
           llvm::count(inputShape, kUnknownSize) != 0) {
         return rewriter.notifyMatchFailure(
-            op,
-            "unimplemented: an inferred dimension is only supported when there "
-            "is enough static shape information to determine its size, or when "
-            "the input tensor is being flattened to a single dimension");
+            op, "unimplemented: an inferred dimension is only supported when "
+                "there is enough static shape information to determine its "
+                "size, or when the input tensor is being flattened to a "
+                "single dimension");
       }
 
       auto productReduceKnownSizes = [](const ArrayRef<int64_t> sizes) {
         auto knownSizes = llvm::make_filter_range(
             sizes, [](int64_t val) { return val != kUnknownSize; });
-        return std::accumulate(knownSizes.begin(), knownSizes.end(), /*init=*/1,
```

Comment:
The formatting with wrapping line changes are mostly not needed. Can you revert those back so that actual changes need are more obvious.